internal/utils/log.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"go.uber.org/zap"
+)
+
+var log *zap.Logger
```

Comment:
See this package: https://github.com/api7/gopkg.

Reply:
already refactor